tests/Resources/ti.ui.ios.navigationwindow.addontest.js

```diff
@@ -41,6 +41,72 @@ describe('Titanium.UI.Window', function () {
 		}
 		win = null;
 	});
+	
+	it.ios('#openWindow', function () {
```

Comment:
FYI, you can just do `describe.ios` at the top to limit all sub-tests to iOS and avoid having to do `it.ios` on them all.

Reply:
Doing now, thx!